aries-site/src/components/button/NavButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, Text } from 'grommet';
+
+export const NavButton = forwardRef(({ active, item, ...rest }, ref) => {
+  const [hover, setHover] = useState(false);
+  return (
+      <Button
+        onMouseOver={() => setHover(true)}
+        onFocus={() => setHover(true)}
+        onMouseOut={() => setHover(false)}
+        onBlur={() => setHover(false)}
+        ref={ref}
+        {...rest}
+      >
+        <Box
```

Comment:
On mobile, I don't think the box should be spanning the full width -- if we apply `align="start"` does that fix it? Or if it doesn't work here maybe doing that on the bounding box of the NavButtons?
<img width="479" alt="Screen Shot 2020-04-02 at 10 17 42 AM" src="https://user-images.githubusercontent.com/12522275/78278737-5915b780-74cb-11ea-9af9-ce7cd5069d3c.png">